src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -367,12 +367,16 @@ class AppStoreConnectCredentialsValidateEndpoint(ProjectEndpoint):  # type: igno
         "itunesSessionValid": true,
         "pendingDownloads": 123,
         "itunesSessionRefreshAt": "YYYY-MM-DDTHH:MM:SS.SSSSSSZ" | null
+        "latestBuildVersion: "9.8.7",
+        "latestBuildNumber": "987000",
     }
     ```
 
     Here the ``itunesSessionRefreshAt`` is when we recommend to refresh the
-    iTunes session, and ``pendingDownloads`` is the number of pending downloads,
-    and an indicator if we do need the session to fetch new builds.
+    iTunes session, and ``pendingDownloads`` is the number of pending build
+    downloads, and an indicator if we do need the session to fetch new builds.
+    ``latestBuildVersion`` is a human-readable equivalent of ``latestBuildNumber``,
+    which is specifically for the latest build that Sentry recognizes.
```

Comment:
```suggestion
    ``latestBuildVersion`` and ``latestBuildNumber`` indicate the last build that
     Sentry recognised on App Store Connect.  It may not yet have downloaded
     symbols for this.
```
The version number and build number **are not** equivalent.  They are a tuple that together identify a build (strictly speaking **maybe** the version number is even irrelevant and it is just the build number.  But treating it as a tuple is on the safe side)